pyproject.toml

```diff
@@ -77,6 +77,7 @@ dependencies = [
     "py-cpuinfo", # display CPU info
     "pandas>=1.1.4",
     "seaborn>=0.11.0", # plotting
+    "shapely>=2.0.6", # zones
```

Comment:
@RizwanMunawar, the security side is good. My only concern here is adding a new dependency.

Reply:
that was required that's why i have added it

Reply:
@Muhammad-237 No, adding this as an external dependency isn't necessary. The Solution class already handles it: https://github.com/ultralytics/ultralytics/blob/a3d807be13af2d8fa008f522020411d2d37e4860/ultralytics/solutions/solutions.py#L51